### ISSUE TITLE
When using a string key for Accept header, the value is overridden by the default

### ISSUE THREAD

**Issue by malcolmsparks** [babashka/http-client#43]
In this case, from the README example, the Accept header is prefixed with `*/*`.

With

```clojure
(http/get "https://httpstat.us/200" {:headers {"Accept" "application/json"}})
```

the actual accept header value is sent as `*/*,application/json`.

As a workaround, this works fine:

```clojure
(http/get "https://httpstat.us/200" {:headers {:accept "application/json"}})
```

the accept header is now `application/json`.

The only difference in the two examples is the use of the keyword `:accept` rather than the string `"Accept"`.

**Comment by borkdude** [comment]
Ah right:

```
user=> babashka.http-client/default-client-opts
{:follow-redirects :normal, :request {:headers {:accept "*/*", :accept-encoding ["gzip" "deflate"], :user-agent "babashka.http-client/0.4.14"}}}
```

Something more clever needs to happen with respect to merging/overriding default options.

**Comment by borkdude** [comment]
@malcolmsparks Fixed on master, need another release?

**Comment by borkdude** [comment]
Released as 0.4.15
